kubernetes/mcp-servers/slack-mcp/slack-deployment.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: slack-mcp-server
+  labels:
+    app: slack-mcp-server
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: slack-mcp-server
+  template:
+    metadata:
+      labels:
+        app: slack-mcp-server
+    spec:
+      containers:
+      - name: slack-mcp-server
+        image: quay.io/rh-ee-anxie/slack-mcp-server:latest
```

Comment:
Can we move this image to the redhat-et quay org? Also, can we add the containerfile as part of this PR too.

Reply:
updated the image, the Containerfile is here: https://github.com/opendatahub-io/llama-stack-demos/blob/main/mcp-servers/slack/Containerfile. We should perhaps figure out how we want to re-organize `mcp-servers` and `kubernetes/mcp-server` (can tackle this in a separate PR)